source/designers-guide/index.html

```diff
@@ -28,11 +28,13 @@
     <li><a href="{{ site.url }}/designers-guide/snippets/">Snippet management</a></li>
     <li><a href="{{ site.url }}/designers-guide/external-resources/">Embedding external resources</a></li>
     <li><a href="{{ site.url }}/designers-guide/responsive-images/">Working with responsive images</a></li>
+    <li><a href="{{ site.url }}/designers-guide/smarty-plugins/">Smarty plugins</a></li>
     <li><a href="{{ site.url }}/designers-guide/best-practice-theme-development/">Using Grunt for theme development</a></li>
     <li><a href="{{ site.url }}/designers-guide/modify-jquery-plugins">Modify jQuery plugins</a></li>
     <li><a href="{{ site.url }}/designers-guide/managing-third-party-dependencies-with-bower/">Managing third-party dependencies with Bower</a></li>
     <li><a href="{{ site.url }}/designers-guide/managing-third-party-dependencies-with-npm/">Managing third-party dependencies with NPM</a></li>
     <li><a href="{{ site.url }}/designers-guide/legacy-template-development-in-shopware-5/">Legacy template development in Shopware 5</a></li>
+    <li><a href="{{ site.url }}/designers-guide/smarty-plugins/">Smarty plugins</a></li>
```

Comment:
The entry is duplicated. Please just add one link in the correct section.

Please add also the link in the same order in `source/designers-guide/general-resources.html`.